wiki/Storyboard_Scripting/zh.md

```diff
@@ -0,0 +1,22 @@
+<!-- TODO: 又来挖坑啦！翻译“规范”丢在故事板里，以后再说 -->
+
+# 利用脚本制作故事板
+
+![](img/SBS_Base.jpg "一个 .osb 文件的故事板脚本例子")
+
+除了内置的[谱面编辑器](/wiki/Beatmap_Editor/Design)以外，也可以用一个纯文本编辑器（例如：[Notepad++](http://www.notepad-plus-plus.org/)甚至是高阶一点的 Visual Studio Code 编辑器）来制作[故事板](/wiki/Storyboards)。这种编辑方法被称为**利用脚本制作故事板**。
+这种方法允许[故事板作者](/wiki/Storyboarding/Storyboarder)能够更精确（以像素或毫秒为单位）地编辑故事板。
```

Comment:
second line doesn't exist in the original, first line seems different

the content needs to be equal

Reply:
The purpose of the Chinese translation group is to translate the original text and supplement the text so that Chinese players can understand the content of the article. It is normal that most articles differ greatly from the original text. Updates to the English text will be given to other members.
中文翻译组的目的是将原文进行翻译并补充让中文玩家能够理解该文章内容，大部分文章与原文有较大出入是正常的。英文原文的更新将交给其他成员。

Reply:
ditto.